File: in_java/Pattern_1.java
package in_java;

public class Pattern_1 extends Pattern {

    public Pattern_1(int row, int column, String symbol) {
        super(row, column, symbol);
    }

    public void print() {
        for (int i = 0; i < this.row; i++) {
            for (int j = 0; j < this.column; j++) {
                if (j <= i) {
                    System.out.print(this.symbol);
                } else {
                    System.out.print(" ");
                }
            }
            System.out.println();
        }
    }
}

File: in_rust/src/pattern_1.rs
pub fn pattern_1() {
    let row = 5;
    let column = 5;
    let symbol = "*";
    for i in 0..row {
        for j in 0..column {
            if j <= i {
                print!("{}", symbol);
            }
        }
        println!();
    }
}
